### PR TITLE
Service status is not working when using init scripts

### PR DESCRIPTION
**Pull request by jlcox1970**
Turns out that the check for the status of the jira server was always returning a state of 'stopped'

This was due to the jps task always having an exit status of 1 even thou the jira service was running.
 have chaged this to simply look for a process owened by the jira user

### CODE REVIEW

File: templates/jira.initscript.erb [hunk 1]
```diff
@@ -64,14 +64,13 @@ function status() {
 
   STATUS=$( ps aux | grep "[c]atalina.base=$CATALINA_HOME" | wc -l )
   if [ $STATUS -gt 0 ];then
-    ps -ef |grep $SERVICE |grep -v grep |awk '{ print $2 }' | <%= scope.lookupvar('jira::javahome') %>/bin/jps |grep -v Jps |grep -v grep > /dev/null
-    RETVAL=$?
-    if [ $RETVAL -eq 0 ];then
+    RETVAL=`ps -ef |grep $SERVICE |grep -v grep |grep atlassian-jira |grep ^<%= scope.lookupvar('jira::user')%> |wc -l`
```
Comment (igalic) [inline templates/jira.initscript.erb]:
```
pgrep -u <%= scope.lookupvar('jira::user')%>  -f atlassian-jira
```

File: templates/jira.initscript.erb [hunk 2]
```diff
@@ -64,14 +64,13 @@ function status() {
 
   STATUS=$( ps aux | grep "[c]atalina.base=$CATALINA_HOME" | wc -l )
```
Comment (igalic) [inline templates/jira.initscript.erb]: this is pretty awful, too…

File: templates/jira.initscript.erb (hunk 1 above)
Comment (igalic) [inline templates/jira.initscript.erb]: come to think of it, this whole set of cascaded ifs could be that line…

**Comment by jyaworski** [comment]
Hello:

What's the status of this PR?

**Comment by bastelfreak** [comment]
HI @jlcox1970, could you take a look at the improvements @igalic mentioned?

**Comment by jlcox1970** [comment]
@bastelfreak I have made some more changes in line of what @igalic was suggesting
My testing shows that it is working as before but the code I think is much cleaner

**Comment by bastelfreak** [comment]
thanks!

**Pull request merged**